lambdatest/README.md

```diff
@@ -25,13 +25,11 @@ All configuration happens on the LambdaTest Dashboard. See the [LambdaTest-Datad
 
 Here's how you can track incidents in Datadog with LambdaTest:
 
-1. Log in to your LambdaTest account on the LambdaTest website.
-2. Select settings from the menu bar.
-3. Click on integrations under settings.
-4. To integrate Datadog, click on install next to Push to Datadog.
-5. Enter your [Datadog API and application key][2].
-6. A confirmation email is sent. Confirm the integration in the email.
-7. Once Datadog is integrated with your LambdaTest account, start logging bugs and performing cross-browser testing.
+1. Click "Connect Accounts" to begin authorization of the LambdaTest integration. You will be taken to the Login page in LambdaTest.
+2. Log in to your LambdaTest account on the LambdaTest website and you'll be redirected to your Datadog application.
+3. Click **Authorize** to complete the integration process.
+4. A confirmation email is sent once the integration configuration is complete.
+5. Once Datadog is integrated with your LambdaTest account, start logging bugs and performing cross-browser testing.
 
```

Comment:
```suggestion

### Uninstallation

To ensure this integration is fully uninstalled, revoke authorization in the Configure tab of this tile or in [OAuth Apps](https://app.datadoghq.com/organization-settings/oauth-applications) within Organization Settings in Datadog. 

Additionally, ensure that all API keys associated with this integration have been disabled by searching for the integration name on the [API Keys management page](https://app.datadoghq.com/organization-settings/api-keys).
```

Please also add these OAuth-specific instructions under the Configuration section. Thanks!